gloo/docs/gloo_routing/virtual_services/security/jwt/claim_routing.md

```diff
@@ -10,13 +10,13 @@ primary version.
 ## Prerequisites
 You will need:
 
-- gloo-e 0.14.0 or higher.
+- gloo-e 0.14.0 or higher installed.
```

Comment:
This is a nit (and applies to the other guide as well), but it would be nice for this section to be consistent with the `Setup` sections we have elsewhere in the auth docs. Maybe we could update that "template" to include utilities that are needed for the guide, e.g. `openssl`, `jq`, etc